Core/Source/Observable/ObservableData.swift

```diff
@@ -1,226 +0,0 @@
-/// Represents a value whose changes can be observed.  Use the standard `Observable`
-/// APIs to observe changes.
-public protocol ObservableData: Observable {
```

Comment:
is anything replacing this and observable variable? they're pretty useful for the non-rx world

Reply:
No I hadn't added anything to replace them, I had figured since they weren't used internally they were a good candidate to remove for simplicity sake (and they'll still lead to some autocomplete confusion/collision, though less so). 

Would you be open to including them in the API just when RxSwift is absent? Or do you think it's confusing to have different APIs?